synapse/lib/storm.lark

```diff
@@ -180,18 +180,20 @@ edgedeln1: _EDGEN1INIT _valu _EDGEN1FINI baresubquery
 edgeaddn2: _EDGEN2INIT _valu _EDGEADDN2FINI baresubquery
 edgedeln2: _EDGEN2INIT _valu _EDGEN2FINI baresubquery
 
-liftformtag: (PROPS | UNIVNAME) tagname [_cmpr _valu]
+liftformtag: (PROPS | UNIVNAME) tagnamenospace [_cmpr _valu]
 _DEREF.3:  /\*(?=\$)/
 liftpropby: ((PROPS | UNIVNAME) | _DEREF _varvalu) _cmpr _valu
 liftprop: ((PROPS | UNIVNAME) | _DEREF _varvalu)
 liftbyarray: ((PROPS | UNIVNAME) | _DEREF _varvalu) "*[" _safe_cmpr _valu "]"
-lifttagtag: "#" tagname [_cmpr _valu]
-liftbytag: tagname [_cmpr _valu]
+lifttagtag:(_HASHSPACE | _HASHNOSPACE) tagnamenospace [_cmpr _valu]
+liftbytag: (tagname | tagnamenospace) [_cmpr _valu]
```

Comment:
Would it be easier to make the lift-by-tag into a single token that we chop up in AST?

Reply:
This feels like it ends up with even more regexing and doesn't actually simplify it.

Reply:
Sorry, i meant lift by form tag.